### ISSUE TITLE
[Accessibility] "Bitwarden has a new look" first-run modal dialog isn't modal

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#12529]
### Steps To Reproduce

1. Do a fresh update/install of the extension
2. Log into/unlock your vault
3. See the "Bitwarden has a new look" dialog

### Expected Result

Initial focus should be set on the modal dialog. Using `Tab`/`Shift+Tab`, focus should be kept inside the modal dialog. The underlying document behind the modal dialog should be hidden from screen readers/assistive technologies.

### Actual Result

Focus is not set into the modal dialog when it's shown.

Focus is not locked to the dialog - a user is able to navigate through the underlying document *behind* the open modal dialog.

The document behind the modal dialog is still exposed to screen readers/assistive technologies.

### Additional Context

![screenshot of the extension showing the first-run modal dialog, but keyboard focus is visible behind the dialog](https://github.com/user-attachments/assets/48dfc6dc-d6b5-4fc7-9d67-36e359430499)

You can see in the screenshot that keyboard focus is behind the modal dialog.

Suspect that this first-run dialog was not triggered properly / didn't do focus management correctly.

### Operating System

Windows

### Operating System Version

11

### Web Browser

Chrome

### Browser Version

131.0.6778.205 (Official Build) (64-bit)

### Build Version

2024.12.2

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SergeantConfused** [comment]
Hello @patrickhlauke,

Thank you for your report. I have flagged this to the Engineering department; please feel free to post additional information, such as screenshots or a screen video recordings, if you wish.

Thank you again,

**Comment by danielleflinn** [comment]
Closing as this dialog is no longer present in the code. https://github.com/bitwarden/test/pull/1613
